src/app/places/places.component.ts

```diff
@@ -1,21 +1,22 @@
 import { Component, OnInit } from '@angular/core';
 import { PlacesService } from '../places.service';
 import { Place } from '../../models/place';
-
+ 
 @Component({
   selector: 'app-places',
   templateUrl: './places.component.html',
   styleUrls: ['./places.component.scss'],
 })
 export class PlacesComponent implements OnInit {
+  p: number = 1;
```

Comment:
Any variable in code should have a name which more or less tell other what it is for.